tests/kubeutils.py

```diff
@@ -0,0 +1,117 @@
+import time
+
+from kubernetes import client
+from kubernetes import watch
+from kubernetes.client import rest
+from kubernetes.stream import stream
+
+
+RESOURCE_READY = 0
+RESOURCE_EMPTY = 1
+RESOURCE_MISSING = 2
+
+
+def assert_resource_availability(resource, node, status, timeout=0):
```

Comment:
If it is the caller who sets the status (explicitly), then it will be more readable to use 3 different asserts, one for each status, something lie: assert_resource_ready()

Reply:
Good point, done.